app/containers/ActionSheet/ActionSheet.tsx

```diff
@@ -96,7 +96,7 @@ const ActionSheet = React.memo(
 
 		const renderHandle = () => (
 			<>
-				<Handle />
+				<Handle backgroundColor={data?.backgroundColor ? data.backgroundColor : colors.focusedBackground} />
```

Comment:
I think we shouldn't give the option to send other colors here.
We should unify the behavior instead, using the same color everywhere.
You can keep the previous color the default one.